src/main/java/practice/StreamPractice.java

```diff
@@ -48,18 +57,56 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      * Example: select people of working age
      * (from 18 y.o. and to 60 y.o. for men and to 55 y.o. for women inclusively).
      */
+
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(getWorkablePredicate(fromAge, femaleToAge, maleToAge))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
You can simplify  it